R/dependencies.R

```diff
@@ -5,11 +5,15 @@
 #' Use in conjunction with the `callback()` method from the [dash::Dash] class
 #' to define the update logic in your application.
 #'
+#' The `dashNoUpdate()` function permits application developers to prevent a
+#' single output from updating the layout. It has no formal arguments, but
+#' returns an object of class `no_update`, for which Dash will return a `204`
+#' status code and empty response body.
```

Comment:
users don't care about the implementation here - they just want to know that as a result the output prop will not change. I'd just drop everything past "It has no formal arguments"

Reply:
fixed in 10404e0